js/packages/web/src/actions/sendRedeemBid.ts

```diff
@@ -819,7 +819,7 @@ export async function setupRedeemParticipationInstructions(
         ? fixedPrice.toNumber()
         : bid?.info.lastBid.toNumber() || 0;
 
-    let tokenAccount = accountsByMint.get(auctionView.auction.info.tokenMint);
+    let tokenAccount = accountsByMint.get(QUOTE_MINT.toBase58());
 
```

Comment:
Curious why we have to make changes here. Shouldn't the auction know its mint?

Reply:
Agree,

well

I couldn't for the life of me figure out why my decimals were wrong on auction card. I still didn't figure it out. Elsewhere on same page it was correct.